website/docs/docs/use-dbt-semantic-layer/sl-cache.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Caching in the dbt Semantic Layer"
+id: "sl-cache"
+description: "Cache common queries to speed up performance and reduce query computation."
+tags: [Semantic Layer]
+sidebar_label: "Cache common queries"
+---
+
+
+The dbt Semantic Layer allows you to cache common queries in order to speed up performance and reduce compute on expensive queries. 
+
+There are two layers of caching:
```

Comment:
can we clarify what could be the use cases to choose one over the other?

Reply:
same use case for both, want queries to be fast. one happens automatically (result) and the other is declarative (delcaring specifically what you want). use case is the same: want to increase performance. in declarative cache, you do have to know or anticipate what users are going to query.